draft-jimenez-t2trg-mud-coap.md

```diff
@@ -129,7 +129,35 @@ TBD : IPv4, coap://, ...
 
 ## Neighbor Discovery Protocol (NDP)
 
-IPv6 hosts do not require DHCP to get access to the default gateway. {{!RFC4861}} can also be used to advertise the MUD URL.
+IPv6 hosts do not require DHCP to get access to the default gateway.
+Using NDP {{!RFC4861}} and Stateless Address Autoconfiguration (SLAAC), nodes can configure global addresses on their own based on prefixes contained in NDP Router Advertisements (RAs).
```

Comment:
I'm unsure whether it's proper etiquette to link to other RFCs everywhere, but since SLAAC has a corresponding RFC ([RFC 4862](https://datatracker.ietf.org/doc/html/rfc4862)), it could be linked here.
```suggestion
Using NDP {{!RFC4861}} and Stateless Address Autoconfiguration (SLAAC) {{!RFC4862}}, nodes can configure global addresses on their own based on prefixes contained in NDP Router Advertisements (RAs).
```

Reply:
Thank you! I incorporated your suggestions :)